straxen/plugins/nveto_recorder.py

```diff
@@ -105,29 +105,37 @@ def compute(self, raw_records_nv, start, end):
                                min_amplitude=self.config['hit_min_amplitude_nv'])
         del temp_records
 
```

Comment:
The resulting raw_records_coin_nv should be identical, but we can also increase the version number for bookkeeping. It will just mean a lot of reprocessing.

Reply:
if you are sure that the results are the same, I stand corrected and no version bump would be needed

Reply:
I checked with a 6 min run (42 million raw_record fragments and 1.6 million raw_records_coin_nv). Shape and all entries are identical.